src/sentry/web/frontend/pipeline_advancer.py

```diff
@@ -35,7 +35,8 @@ def handle(self, request, provider_id):
             if pipeline:
                 break
 
-        if provider_id in FORWARD_INSTALL_FOR and request.GET.get('setup_action') == 'install':
+        if provider_id in FORWARD_INSTALL_FOR and request.GET.get(
+                'setup_action') == 'install' and pipeline is None:
```

Comment:
Kind of a weird place to break the line, imo. Is that a normal python way of doing multiline conditionals?

Reply:
i had the same thought tbh. the commit hook did this.

Reply:
Ahh, yea, it breaks things weirdly sometimes. 😒